tests/build.rs

```diff
@@ -14,12 +14,40 @@ fn it_builds_webpack() {
     let fixture = Fixture::new();
     fixture.scaffold_webpack();
 
-    let wrangler_toml = WranglerToml::webpack_no_config("test-build-webpack");
+    let wrangler_toml = WranglerToml::webpack_build("test-build-webpack");
     fixture.create_wrangler_toml(wrangler_toml);
 
     build_creates_assets(&fixture, vec!["script.js"]);
 }
 
+#[test]
+fn it_builds_webpack_site() {
+    let fixture = Fixture::new_site();
+
+    let wrangler_toml = WranglerToml::site("test-build-site");
+    fixture.create_wrangler_toml(wrangler_toml);
+
+    build_creates_assets(&fixture, vec!["main.js"]);
+}
+
+#[test]
+fn it_builds_webpack_site_with_custom_webpack() {
+    let fixture = Fixture::new_site();
+
+    fixture.create_file(
+        "workers-site/webpack.worker.js",
+        r#"
+        module.exports = { context: __dirname, entry: "./index.js" };
```

Comment:
should `target: webworker` in case somebody tries to copy our test suite instead of looking at our docs :)

Reply:
i... O.o

Reply:
should we do that for all of them though? because none of the rest of our tests do this.

Reply:
should we do that for all of them though? because none of the rest of our tests do this.

Reply:
hmm yeah we probably should... if we dont provide a custom config then it should be using it automatically